### PR TITLE
Shalamai.week2

### PR DESCRIPTION
**Pull request by shalamai**
all done except updateContactInfo()

---
This change is [Reviewable](https://reviewable.io/reviews/presly808/aco17/12)

### CODE REVIEW

**Comment by shalamai** [comment]
check, please

File: VladShalamai/src/contactlist/Contact.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
+    public static int getNumberOfContact() {
+        return numberOfContact;
+    }
+
+    public static void setNumberOfContact(int numberOfContact) {
```
Comment (presly808) [inline VladShalamai/src/contactlist/Contact.java]: unnecessary method, you may remove it

File: VladShalamai/src/contactlist/ContactList.java [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+        return sb.toString();
+
+    }
+
+    public String showFirstFiveContacts() {
+
+        StringBuilder sb = new StringBuilder();
+
+        for (int i = 0; i < 5; i++) {
+            if (contacts[i] != null) {
```
Comment (presly808) [inline VladShalamai/src/contactlist/ContactList.java]: You should to keep some variable that will know about actual size of elements excluding null values
It simplify adding and showing by removing if(not null) condition

File: VladShalamai/src/contactlist/TestContact.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+        testShowInfo(contact);
+
+    }
+
+    public static void testShowInfo(Contact contact) {
+        String actual = contact.showInfo();
+        String expected = "name is George, email: george.@gmail.com, phoneNumber: 095 895 34 23";
+
+        System.out.println("menthod showInfo() is " + expected.equals(actual));
```
Comment (presly808) [inline VladShalamai/src/contactlist/TestContact.java]: use one println(printf) to show the test's assertion result

File: VladShalamai/src/university/SubjectUtils.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
+package university;
+
+import week2.CarUtils;
+
+public class SubjectUtils {
```
Comment (presly808) [inline VladShalamai/src/university/SubjectUtils.java]: You can merge utils class to the one common

**Comment by shalamai** [comment]
is corrected

**Pull request merged**